src/util/NuulsUploader.cpp

```diff
@@ -62,20 +64,24 @@ void uploadImageToNuuls(RawImageData imageData, ChannelPtr channel,
         .header("Content-Type", contentType)
 
         .multiPart(payload)
-        .onSuccess([&textEdit, channel](NetworkResult result) -> Outcome {
+        .onSuccess([&textEdit, channel,
+                    originalFilePath](NetworkResult result) -> Outcome {
             textEdit.insertPlainText(result.getData() + QString(" "));
             if (uploadQueue.empty())
             {
                 channel->addMessage(makeSystemMessage(
-                    QString("Your image has been uploaded.")));
+                    QString("Your image has been uploaded to ") +
+                    result.getData()));
                 uploadMutex.unlock();
             }
             else
             {
                 channel->addMessage(makeSystemMessage(
-                    QString("Your image has been uploaded. %1 left. Please "
-                            "wait until all of them are uploaded. About %2 "
-                            "seconds left.")
+                    QString(
+                        "Your image has been uploaded to %1 %2 left. Please "
+                        "wait until all of them are uploaded. About %3 "
+                        "seconds left.")
+                        .arg(result.getData() + QString(""))
```

Comment:
Re-add the . after the "to %1" part

Reply:
Re-added with a space between, because without a space before the dot, links are invalid.